pkg/kubelet/dockertools/manager.go

```diff
@@ -1643,8 +1620,8 @@ func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubetypes.Docker
 		return "", err
 	}
 
-	// There is no meaningful labels for infraContainer now, so just pass nil.
-	id, err := dm.runContainerInPod(pod, container, netNamespace, getIPCMode(pod), getPidMode(pod), nil)
+	// Current now we don't care about restart count of infra container, just set it to 0.
```

Comment:
Ops...I don't even know where I learned it. I really need more suggestions like this about my grammar, words etc. Thank you very much! :)